packages/ibm-products/src/components/EmptyStates/EmptyState.tsx

```diff
@@ -116,6 +121,7 @@ export let EmptyState = React.forwardRef<HTMLDivElement, EmptyStateProps>(
       illustrationPosition = defaults.position,
       link,
       size = defaults.size,
+      headingAs,
```

Comment:
We can give a default value for `headingAs ` as 'h3', since currently we have this as h3